frontend/src/fixtures/schoolsFixtures.js

```diff
@@ -3,30 +3,26 @@ const schoolsFixtures = {
         "abbrev": "ucsb",
         "name": "UC Santa Barbara",
         "termRegex": "[WSMF]\\d\\d",
-        "termDescription": "Enter quarter, e.g. F23, W24, S24, M24",
-        "termError": "Quarter must be entered in the correct format"
+        "termDescription": "Enter quarter, e.g. F23, W24, S24, M24"
```

Comment:
```suggestion
        "termDescription": "quarter"
```

Let's just go with either "quarter" or "semester" in these examples.